packages/coil-extension/src/popup/components/HeaderFooterLayout.tsx

```diff
@@ -21,8 +21,7 @@ import { Container } from './util/Container'
 const OuterDiv = styled('div')({
   minWidth: '308px',
   maxWidth: '308px',
-  height: 'auto',
-  minHeight: '260px'
```

Comment:
![image](https://user-images.githubusercontent.com/525211/122523625-72a80e00-d041-11eb-9001-ac3cbf3fe6f2.png)

@mwill8886  
Can you weigh in on this?

Reply:
@sharafian 

Chatted to @mwill8886 out of band, and he's approved this change